Add init scripts and use the correct group name

bruj0 opened the pull request:
Hi, this change adds an init file for Centos based distributions and a bug fix for using the correct group name.

bshi commented:
Hi @bruj0, I'm happy you found this useful!  I'm not really using this or maintaining it any more - so if you have a fork you're willing to maintain, I'm happy to just update this repo with a deprecation notice that points to your fork.

bruj0 commented:
@bshi Thank you for starting the project. It will be my pleasure to maintain it going forward.
I also implemented /etc/group support and added an extra group without sudo access.
